easy_rec/python/model/easy_rec_model.py

```diff
@@ -304,3 +304,22 @@ def get_grouped_vars(self):
        grouped_vars: list of list of variables
     """
     raise NotImplementedError()
+
+  def _build_default_output_names(self):
+    return [ 'feature_' + name for name in self._feature_dict ]
+
+  def _build_default_outputs(self):
+    for feature_name in self._feature_dict:
```

Comment:
what is the purpose of this function?

Reply:
Output the original features fed into the model_fn. It's useful for debugging.